src/Sylius/Bundle/ResourceBundle/spec/Form/DataTransformer/IdentifierToResourceTransformerSpec.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\ResourceBundle\Form\DataTransformer;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+// Since the root namespace "spec" is not in our autoload
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'FakeEntity.php';
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class IdentifierToResourceTransformerSpec extends ObjectBehavior
+{
+    function let(RepositoryInterface $repository)
+    {
+        $repository->getClassName()->willReturn('spec\Sylius\Bundle\ResourceBundle\Form\DataTransformer\FakeEntity');
```

Comment:
`FakeEntity::class`, anyway, the example alone should be readable, so I think this line should be included in specific examples.